### DISCUSSION TITLE
pixi-actions (npm library)

### DISCUSSION THREAD

**Discussion by srpatel** [pixijs/pixijs#7293] [Show and tell]
pixi-actions is a little library for combining tweens/animations of DisplayObjects. It's inspired by the LibGDX action system, and follows a very similar API.

For example, to animate a sprite to the right, fading it out as it goes, and removing from the parent at the end:

    Actions.parallel(
        Actions.moveTo(sprite, sprite.x + 100, sprite.y, 5),
        Actions.fadeOutAndRemove(sprite, 5)
    ).play();

Github: https://github.com/srpatel/pixi-actions#readme

I made this for a game jam I participated in last week, and I'm certainly going to be using it again. I thought it might be helpful for other people, especially anyone coming from LibGDX land.

**Comment by ivanpopelyshev** [comment]
Its strange that we didnt have this sub-section before: https://github.com/pixijs/pixi.js/wiki/v5-Resources#tweens--animation
